core/src/advice/inputs.rs

```diff
@@ -132,7 +131,7 @@ impl AdviceInputs {
 
     /// Decomposes these `[Self]` into their raw components.
     #[allow(clippy::type_complexity)]
-    pub(crate) fn into_parts(self) -> (Vec<Felt>, AdviceMap, MerkleStore) {
+    pub fn into_parts(self) -> (Vec<Felt>, AdviceMap, MerkleStore) {
```

Comment:
Do we still need `#[allow(clippy::type_complexity)]` here?

Reply:
No, removed